modules/oc-mirror-enclave-support.adoc

```diff
@@ -102,6 +102,13 @@ $ export UPDATE_URL_OVERRIDE=https://<osus.enterprise.in>/graph
 +
 For more information on setting up OSUS on an OpenShift cluster, see "Updating a cluster in a disconnected environment using the OpenShift Update Service".
 
+[NOTE]
+====
+When upgrading {product-title} Extended Update Support (EUS) versions, an intermediate version may be needed between the current and target versions. For example, if the current version is `4.14` and target version is `4.16`, you may need to include a version like `4.15.8` in the `ImageSetConfiguration` when using oc-mirror plugin v1.
```

Comment:
Enclave support means v2 right, why do we mention about v1 here ?

Reply:
It is v2, I will change it.